repo_guidelines.md

```diff
@@ -49,13 +55,39 @@ See our [proposed linting standard](./protocol/linting.md) for further details.
 
 ### Minimal Contributing Guidelines
 
-```
+```md
 # Contributing Guidelines
 
-We love improvements to our tools! EDGI has general [guidelines for contributing](https://github.com/edgi-govdata-archiving/overview/blob/master/CONTRIBUTING.md) and a [code of conduct](https://github.com/edgi-govdata-archiving/overview/blob/master/CONDUCT.md) for all of our organizational repos.
+We love improvements to our tools! EDGI has general [guidelines for contributing][edgi-contributing] and a [code of conduct][edgi-conduct] for all of our organizational repos.
+
+## Here are some notes specific to this project:
+Use this section for any contributing guidelines specific to the repo. For example:
+* [Web monitoring's processing repo guidelines](https://github.com/edgi-govdata-archiving/web-monitoring-processing/blob/master/CONTRIBUTING.md) mentions which issues should be made on that repo versus on the main web monitoring repo
+* In the [Web Monitoring umbrella repo's contribution guidelines](https://github.com/edgi-govdata-archiving/web-monitoring/blob/master/CONTRIBUTING.md), a style guide, testing best practices, and notes on how work is distributed (process) are laid out
```

Comment:
This section is meant as a set of examples (see line 64), not to be copy/pasted into every contributing.md. What would be a good way to make that clearer?

Reply:
Maybe calling this whole thing an example and not a template? Template tells me I should copy this code snippet into a file and fill in the blanks.

Reply:
Well, now that I’ve seen what this looks like with the bullet points moved out of the code sample, I feel like an idiot, because I feel like it reads less clearly than what you originally had.

Maybe if this the original introductory sentence was shorter, had some white space around it to be more visible, and square brackets?

```md
## Here are some notes specific to this project:

**[REPLACE WITH REPO-SPECIFIC GUIDELINES. The list below is an example.]**

* [Web monitoring's processing repo guidelines](https://github.com/edgi-govdata-archiving/web-monitoring-processing/blob/master/CONTRIBUTING.md) mentions which issues should be made on that repo versus on the main web monitoring repo
* In the [Web Monitoring umbrella repo's contribution guidelines](https://github.com/edgi-govdata-archiving/web-monitoring/blob/master/CONTRIBUTING.md), a style guide, testing best practices, and notes on how work is distributed (process) are laid out...
* etc.
```

Otherwise I guess the way you originally had it feels best, and I am just a bad reader.

Reply:
no worries, workshopping is a good exercise! I like your latest, will switch to that.